src/Components/Card/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import { CardProps } from "./types";
+import Typography from "../Typography/Typography";
+import { TypographyTypes } from "../Typography/consts";
+import { colors } from "../../global-styles";
+import {
+  StyledCard,
+  StyledCardActionArea,
+  StyledImg,
+  ImgContainer,
+  ContentContainer,
+  IconContainer,
+  PowerText,
+} from "./styles";
+import StrengthIcon from "./strengthIcon";
+
+const Card = ({ id, img, name, power }: CardProps) => {
+  return (
+    <StyledCard>
+      <StyledCardActionArea>
+        <ImgContainer>
+          <StyledImg src={img} />
```

Comment:
provide a fallback text for the `alt` prop.
what happens when the url is broken ? what is rendered to the screen ?